packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -227,12 +227,18 @@ class ScrollDragController implements Drag {
     @required DragStartDetails details,
     this.onDragCanceled,
     this.carriedVelocity,
+    this.motionStartDistanceThreshold
   }) : assert(delegate != null),
        assert(details != null),
+       assert(
+         motionStartDistanceThreshold == null || motionStartDistanceThreshold > 0.0,
+         'Must be a positive number or null'
```

Comment:
This should say _what_ must be positive or null. The error message will have no context.

Reply:
Seems like asserts with optional messages actually shows both the assert test code and the message.

Reply:
Yes, but experience shows our users don't understand the assertion test code. They only read the text (if we're lucky). (The framework actually flips the message around so the message comes first and the assert code is second, now.)

Reply:
Done